package.json

```diff
@@ -3,16 +3,18 @@
   "version": "0.23.1",
   "description": " React port of slick carousel",
   "main": "./lib",
-  "files": ["dist", "lib"],
+  "files": [
+    "dist",
+    "lib"
+  ],
   "scripts": {
     "start": "gulp server",
     "build": "gulp clean && gulp sass && gulp copy && webpack",
     "prepublish": "babel ./src --out-dir ./lib && gulp dist",
-    "test": "eslint src && jest",
+    "test": "eslint src && jest --updateSnapshot",
```

Comment:
@maddhruv We shouldn't update snapshot automatically.

If some tests, we should update snapshots if necessary in interactive mode

Reply:
@maddhruv remove --updateSnapshot option